server/server.js

```diff
@@ -1,16 +1,43 @@
 const express = require('express');
+const querystring = require('querystring');
```

Comment:
Is querystring actually being used in here? Express should handle any querystring stuff.